### ISSUE TITLE
Compile error installing from package.el

### ISSUE THREAD

**Issue by jjengla** [emacsorphanage/terraform-mode#36]
I tried installing **terraform-mode-20201110.1915** via package.el and get the following error:
`terraform-mode.el:71:1:Error: rx form ‘regexp’ requires args satisfying ‘stringp’`

**Comment by rnikoopour** [comment]
What version of emacs are you using and which OS?

I cannot reproduce this on Emacs 27.1 on OSX.

I tried removing my `~/.emacs.d` folder and only had terraform configured to install on startup in my `~/.emacs`.  This did not reproduce the error you received.

I also tried removing all package installations from my `~/.emacs` and installed `terraform-mode` version `20201110.1915` from `package-list-packages` manually.  This did not reproduce the error you received either.

**Comment by opsaaspo** [comment]
I have the same error.
The emacs info is
```
GNU Emacs 26.3 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.24.14) of 2020-03-26, modified by Debian
```

**Comment by rnikoopour** [comment]
Thank you I was able to reproduce this using `GNU Emacs 26.3` inside an Ubuntu container.

I'll work on getting a patch out for this by the end of this weekend.

**Comment by rnikoopour** [comment]
@opsaaspo @jjengla Could you please try using `terraform-mode.el` from #38. 

I was able to byte-compile the file using `GNU Emacs 26.3` on an Ubuntu container.

<img width="1067" alt="emacs26_ubuntu_container" src="https://user-images.githubusercontent.com/7692789/99039216-d0202d80-253b-11eb-851c-a8beb9ba3bbf.png">

**Comment by mstg** [comment]
@rnikoopour I am not the original reporter, but #38 is working for me on `GNU Emacs 26.3` on macOS. Thanks!

**Comment by rnikoopour** [comment]
Fixed in #38
